src/ontology/phenio-odk.yaml

```diff
@@ -65,41 +66,35 @@ import_group:
       mirror_from: http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-fbbt.owl
     - id: cl-bridge-to-zfa
       mirror_from: http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-zfa.owl
+#    - id: sepio
+#      make_base: TRUE
 components:
   products:
     - filename: pato.owl
       source: http://purl.obolibrary.org/obo/pato/pato-base.owl
     - filename: go.owl
       source: http://purl.obolibrary.org/obo/go/go-base.owl
-      ###
-      # upheno2
-    - filename: upheno.owl
+    - filename: upheno.owl # upheno2
       source: https://bbop-ontologies.s3.amazonaws.com/upheno/current/upheno-release/all/upheno_all_with_relations.owl
-      ###
-    - filename: mondo-equivalents.owl
-      source: http://purl.obolibrary.org/obo/mondo/imports/equivalencies.owl
+#    - filename: mondo-equivalents.owl
+#      source: http://purl.obolibrary.org/obo/mondo/imports/equivalencies.owl
     - filename: mondo.owl
-      source: http://purl.obolibrary.org/obo/mondo/mondo-base.owl
+      source: https://github.com/monarch-initiative/mondo/releases/download/v2022-08-01/mondo-base.owl
```

Comment:
Lets not merge this change - this was a temporary outtage.

Reply:
Good catch. Reverted.